migration/secure_note_transformer.py

```diff
@@ -0,0 +1,206 @@
+            },
+            {
+                "id": "validFrom",
+                "type": "MONTH_YEAR",
+                "label": "valid from",
+                "value": utils.lpass_date_to_1password_format(data["Start Date"])
+            },
+        ]
+
+    def _map_bank_account(self, data, template):
+        template["title"] = self.lpass_raw_data.title
+        template["fields"] = [
+            {
+                "id": "notesPlain",
+                "type": "STRING",
+                "purpose": "NOTES",
+                "label": "notesPlain",
+                "value": data["Notes"]
+            },
+            {
+                "id": "bankName",
+                "type": "STRING",
+                "label": "bank name",
+                "value": data["Bank Name"]
+            },
+            {
+                "id": "accountType",
+                "type": "MENU",
+                "label": "type",
+                "value": data["Account Type"]
+            },
+            {
+                "id": "routingNo",
+                "type": "STRING",
+                "label": "routing number",
+                "value": data["Routing Number"]
+            },
+            {
+                "id": "accountNo",
+                "type": "STRING",
+                "label": "account number",
+                "value": data["Account Number"]
+            },
+            {
+                "id": "swift",
+                "type": "STRING",
+                "label": "SWIFT",
+                "value": data["SWIFT Code"]
+            },
+            {
+                "id": "iban",
+                "type": "STRING",
+                "label": "IBAN",
+                "value": data["IBAN Number"]
+            },
+            {
+                "id": "telephonePin",
+                "type": "CONCEALED",
+                "label": "PIN",
+                "value": data["Pin"]
+            },
+            {
+                "id": "branchPhone",
+                "section": {
+                    "id": "branchInfo",
+                    "label": "Branch Information"
+                },
+                "type": "PHONE",
+                "label": "phone",
+                "value": data["Branch Phone"]
+            },
+            {
+                "id": "branchAddress",
+                "section": {
+                    "id": "branchInfo",
+                    "label": "Branch Information"
+                },
+                "type": "STRING",
+                "label": "address",
+                "value": data["Branch Address"]
+            }
+        ]
+
+
+def fetch_template(item_type: str):
+    try:
+        template = subprocess.run([
+            "op", "item", "template", "get", item_type,
+            "--format=json"
+        ], check=True, capture_output=True)
+        return json.loads(template.stdout)
+    except:
+        logging.warning(f"An error occurred when attempting to fetch the {item_type} item template.")
```

Comment:
Can't we use this functionality to fetch the login template as well?

Reply:
Yes, nice suggestion. I changed it to do so.